components/HackneyDS/Dialog/index.js

```diff
@@ -13,9 +13,15 @@ export default function Dialog({
   const [windowState, setWindowState] = useState();
 
   useEffect(() => {
-    document.querySelector('html').style.overflow = isOpen ? 'hidden' : 'visible';
+    document.querySelector('html').style.overflow = isOpen ? 'hidden' : '';
   }, [isOpen]);
 
+  useEffect(() => {
+    return () => {
+      document.querySelector('html').style.overflow = '';
```

Comment:
I don't know how this even snuck into the project but can you please remove querySelectors use from this file.